index.html

```diff
@@ -0,0 +1,42 @@
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>simpleweb</title>
+    </head>
+    <body>
+        <header>
+        <div class="toolbar">
+            <div class="logo">My Logo</div>
+            <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+        </div>
+        </header>
+        <div class="img1"><img src="images/elli1.png" alt="ellipse" width="24" height="28"></div>
+        <div class="img2"><img src="images/rect.png" width="24" height="21"></div>
+        <div class="img3"><img src="images/el2.png" ></div>
+        <img class="img4" src="images/el3.png" >
+    <section>
+         <div> 
+        <h1 class="header">I'm Mariane<br>Glad to see you!</h1></div>    
+      <div>  <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    </section>
+        <footer class="contact">
```

Comment:
- [ ] Kindly don't use the `<footer/>`  tag at the moment because you will use it later. Currently, you are working on the headline section of the page.